### PR TITLE
Add mailerlite TXT and SPF records to sign mail

### PR DESCRIPTION
**Pull request by RhinosF1**

### CODE REVIEW

**Comment by RhinosF1** [comment]
Assuming this is right:
System gives me

TXT Name: ml._domainkey.miraheze.org

TXT Value: k=rsa; p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDJkg+PZpFMFrzwY4ek66rO8OzbkSEuqcK+dDl2Jrla6HoEZLV0Cmwl4tLpFXJN7qgPeG/6yImZ8YOVhMoft08QAuQAaxVcUuZg8D+50DPkFh8tXkWyAt8A2bBzrvd2MpMnHNK6MfqAbAP9OaX/gQmsM3SIpoiwtGCIeQEhqbz8JQIDAQAB

And
TXT Record Value: v=spf1 include:_spf.mlsend.com include:_spf.google.com ip4:185.52.1.76 ip6:2a00:d880:6:787::3 -all

**Comment by Southparkfan** [comment]
Doesn't this allow a third-party to send mail in Miraheze's name with our root domain? Can't we restrict this further, ie. *@lists.miraheze.org?

**Comment by RhinosF1** [comment]
> Doesn't this allow a third-party to send mail in Miraheze's name with our root domain? Can't we restrict this further, ie. *@lists.miraheze.org?

If you'd rather, the system is controlled by us and we control who can send campaigns but in theory they could.

File: zones/miraheze.org [hunk 1]
```diff
@@ -30,10 +30,11 @@ list		MX	10	mail.miraheze.org.
 
 ; Mail policies
 ;; SPF - Policy: Mail not from misc1 is rejected.
-miraheze.org.	TXT	"v=spf1 include:_spf.google.com ip4:185.52.1.76 ip6:2a00:d880:6:787::3 -all"
+miraheze.org.	TXT	"v=spf1 include:_spf.mlsend.com include:_spf.google.com ip4:185.52.1.76 ip6:2a00:d880:6:787::3 -all"
 
 ;; DKIM - Policy: Mail may be signed. Not required.
 mail._domainkey	TXT	"v=DKIM1; k=rsa; p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDLIGrjEMDm0fgER69Wv3OQEPmVPpzfbvspXe+FjFyHnZ2kbuQhMHjDgwX6E/tA5NzDG+dhgaUV0D90H12dhu3wXWaocyLjxrAs9dNkqK69hG8oLdDj8KFpPI6JAm6LskBXHmlOpdzRc+yNBRm5wT9fIsw8in7fnAW170Jb3fdEbwIDAQAB"
+ml._domainkey.miraheze.org TXT "k=rsa; p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDJkg+PZpFMFrzwY4ek66rO8OzbkSEuqcK+dDl2Jrla6HoEZLV0Cmwl4tLpFXJN7qgPeG/6yImZ8YOVhMoft08QAuQAaxVcUuZg8D+50DPkFh8tXkWyAt8A2bBzrvd2MpMnHNK6MfqAbAP9OaX/gQmsM3SIpoiwtGCIeQEhqbz8JQIDAQAB"
```
Comment (JohnFLewis) [inline zones/miraheze.org]: ml._domainkey.miraheze.org.miraheze.org?

File: zones/miraheze.org [hunk 2]
```diff
@@ -30,10 +30,11 @@ list		MX	10	mail.miraheze.org.
 
 ; Mail policies
 ;; SPF - Policy: Mail not from misc1 is rejected.
-miraheze.org.	TXT	"v=spf1 include:_spf.google.com ip4:185.52.1.76 ip6:2a00:d880:6:787::3 -all"
+miraheze.org.	TXT	"v=spf1 include:_spf.mlsend.com include:_spf.google.com ip4:185.52.1.76 ip6:2a00:d880:6:787::3 -all"
```
Comment (JohnFLewis) [inline zones/miraheze.org]: Prefer domain delegation.

**Comment by RhinosF1** [comment]
@JohnFLewis sent you new values on IRC for lists.miraheze.org

**Pull request merged**